README.md

```diff
@@ -25,13 +25,13 @@ npm install --save-exact @holochain/conductor-api
 ## Sample usage
 
 ### Use AdminWebsocket
-```
+```javascript
   const admin = await AdminWebsocket.connect(`http://localhost:8000`, TIMEOUT)
   await admin.generateAgentPubKey()
 ```
 
 ### Use AppWebsocket
-```
+```javascript
   const signalCb = (signal: AppSignal) => {
```

Comment:
This isn't javascript! :) I'm surprised it doesn't fail to parse
```suggestion
```typescript
  const signalCb = (signal: AppSignal) => {
```